### ISSUE TITLE
Update "Workgroups" dropdown filter on "Projects" page

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#12349]
![Screenshot 2023-05-23 at 9 37 41 AM](https://github.com/cityofaustin/atd-data-tech/assets/1463708/5046a3bc-5649-4886-84fd-83e58ba1e5bd)

**Comment by amenity** [comment]
@johnclary - Are these being pulled from `Workgroup:` labels? If so... I wonder if we could the label _description_ instead of the label _name_.

**Comment by johnclary** [comment]
Yes, they are parsed out of the workgroup labels. This actually happens during the Socrata ETL, and it should be possible to do what you’re describing. I’d size it as a 3 depending on if another API call is needed.

https://github.com/cityofaustin/atd-service-bot/blob/production/issues_to_socrata.py

**Comment by amenity** [comment]
Many years later, it's not useful to do this since GitHub has issue fields. As soon as they support multi-select, we'll want to remap to a custom division name instead of a label.
